kolibri/utils/file_transfer.py

```diff
@@ -36,7 +72,7 @@ def __init__(
         block_size=2097152,
         remove_existing_temp_file=True,
         timeout=DEFAULT_TIMEOUT,
-        cancel_check=None,
+        cancel_check=lambda: False,
```

Comment:
what's the advantage of this change in the default value? I only see cancel_check being used in `if` sentences, so None works fine there too